openforcefield/topology/topology.py

```diff
@@ -46,49 +47,65 @@
 #=============================================================================================
 
 # TODO: Can we have the `ALLOWED_*_MODELS` list automatically appear in the docstrings below?
-# TODO: Should these be the Python equivalent of enumerated types.
+# TODO: Should `ALLOWED_*_MODELS` be objects instead of strings?
+# TODO: Should these be imported from `openforcefield.cheminformatics.aromaticity_models` and `.bondorder_models`?
 
-DEFAULT_AROMATICITY_MODEL = 'MDL' # TODO: Is there a more specific spec for this?
+DEFAULT_AROMATICITY_MODEL = 'MDL' # TODO: Is there a more specific name and reference for the aromaticity model?
 ALLOWED_AROMATICITY_MODELS = ['MDL']
 
-DEFAULT_FRACTIONAL_BONDORDER_MODEL = 'Wiberg' # TODO: Is there a more specific spec for this?
+DEFAULT_FRACTIONAL_BONDORDER_MODEL = 'Wiberg' # TODO: Is there a more specific name and reference for the aromatciity model?
 ALLOWED_FRACTIONAL_BONDORDER_MODELS = ['Wiberg']
 
-DEFAULT_CHARGE_MODEL = 'AM1-CM2' # TODO: Should this be `AM1-BCC`?
+DEFAULT_CHARGE_MODEL = 'AM1-CM2' # TODO: Should this be `AM1-BCC`, or should we encode BCCs explicitly?
```

Comment:
Default should be AM1-BCC. Optionally one could specify their own BCCs, but we should just continue relying on OE for this.